### PR TITLE
Display the file drop message when there are existing apps

### LINKED ISSUE #2284: Dragging local charms over canvas doesn't show message if canvas isn't empty
- Add a local charm to an empty canvas (you'll see the drag message and the blue line at the bottom of the window)
- Add a charm to the canvas
- Drag over the canvas again, you'll see that the message is gone

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2284

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- hover a charm zip on the canvas
- you should see the drop message
- drop the charm and upload
- hover the charm zip on the canvas again
- you should see the drop message

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6337/
Test PASSed.

File: jujugui/static/gui/src/app/assets/css/canvas/_onboarding.scss [hunk 1]
```diff
@@ -20,11 +20,27 @@
   }
 
   &__drop-message {
-    color: $warm-grey;
-    font-size: 32px;
-    opacity: 0;
-    position: relative;
-    top: 230px;
+      position: absolute;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/assets/css/canvas/_onboarding.scss]: Looks like 4 spaces.

**Comment by anthonydillon** [comment]
## QA issue
- Drag a random (none juju zip) onto the canvas
- An error notification appears but the environment drop message does not disappear

default juju gui

**Comment by hatched** [comment]
@anthonydillon  looks like that issue is pre-existing. I created a new issue https://github.com/juju/juju-gui/issues/2295

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6577/

**Comment by huwshimi** [comment]
Should I fix that issue as part of this branch, or land this one first?

**Comment by frankban** [comment]
I'd be +1 on fixing the pre-existing issue in a follow up branch.

**Comment by anthonydillon** [comment]
Good to go with a follow up branch.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**